includes/admin/class-amp-post-meta-box.php

```diff
@@ -167,7 +167,7 @@ public function render_status( $post ) {
 		$verify = (
 			isset( $post->ID )
 			&&
-			is_post_type_viewable( $post->post_type )
+			post_type_supports( $post->post_type, amp_get_slug() )
```

Comment:
I'm not sure if this is the right approach. It would mean that a user would never be presented with the prompt to enable the post type support as shown here:

https://github.com/Automattic/amp-wp/blob/4f2f8dd09a974a6ddf4c28b21e41aa341c79ca0c/templates/admin/amp-status.php#L49-L52